src/Tooltip/Tooltip.tsx

```diff
@@ -84,8 +85,11 @@ export class Tooltip extends React.Component<TooltipProps, TooltipState> {
 
     onTooltipContentBlur = (e: React.FocusEvent<HTMLDivElement>): void => {
         const triggeredNode: Node = (e.relatedTarget as Node) || document.activeElement;
-        if (this.props.trigger !== "click" || !document.getElementById(this.state.referenceId).contains(triggeredNode)) {
+        const isWithinTriggerNode: boolean = document.getElementById(this.state.referenceId).contains(triggeredNode);
```

Comment:
Is there any reason why you are using documentById here despite the fact that you have the same element using ref ?

Reply:
no reason, the ref wasn't there in the beginning, it was added later. I replaced getElementById with ref. Nice catch!